compute_tools/src/compute.rs

```diff
@@ -1367,6 +1368,46 @@ LIMIT 100",
         download_size
     }
 
+    pub async fn set_role_grants(
+        &self,
+        db_name: &str,
+        schema_name: &str,
+        privileges: &[Privilege],
+        role_name: &str,
+    ) -> Result<()> {
+        use tokio_postgres::config::Config;
+        use tokio_postgres::NoTls;
```

Comment:
Any reason you didn't import these at the top-level? This is not a pattern I've seen in the code.

Reply:
conflicts with the other import of `postgres::Config` used elsewhere in the file for non-async code.